packages/react-compass/src/slider/slider.tsx

```diff
@@ -63,17 +63,27 @@ const Slider = React.forwardRef<HTMLDivElement, SliderProps>((props, ref) => {
   useEffect(() => {
     const slider = sliderRef.current
     if (slider) {
-      const thumb = slider.querySelector('.thumb') as HTMLElement
-      const sliderProgress = slider.querySelector(
-        '.range-slider',
-      ) as HTMLElement
+      const thumb = slider.querySelector('.cdg-thumb') as HTMLElement
+      const sliderProgress = slider.querySelector('.cdg-track') as HTMLElement
       let isDragging = false
       let prevX = 0
 
-      const handleMouseDown = (event: MouseEvent) => {
+      const handleMouseDown = (event: MouseEvent | TouchEvent) => {
```

Comment:
Slider changes should be placed in other pull request.
Please don't try to push everything into single pull request that for the other purpose.